redbot/core/modlog.py

```diff
@@ -508,7 +511,11 @@ def __init__(
         self.case_str = case_str
         self.guild = guild
         if kwargs:
-            log.warning("Got unexpected keys in case %s", ",".join(kwargs.keys()))
+            log.warning(
+                "Fix this using the hidden command: `modlogset fixcasetypes` in Discord: "
```

Comment:
This command only fixes the problem if unexpected key is `audit_type`, right?

Reply:
Right now, yes. But that is the only unexpected key possible to exist there currently. I left this generic so that it doesn't need changing in the future.

Reply:
I think it would actually be better to change it to just say it on `audit_type`, because this command won't solve the issue about some other unexpected key unless it actually gets updated appropriately and when such change happens, this error can easily be updated for this too.